editor/themes/editor_theme_manager.cpp

```diff
@@ -282,9 +282,15 @@ EditorThemeManager::ThemeConfiguration EditorThemeManager::_create_theme_config(
 			Color preset_base_color;
 			float preset_contrast = 0;
 			bool preset_draw_extra_borders = false;
+			float preset_icon_saturation = config.icon_saturation;
 
 			// Please use alphabetical order if you're adding a new theme here.
-			if (config.preset == "Breeze Dark") {
+			if (config.preset == "Android") {
+				preset_accent_color = Color(0.34, 0.62, 1.00);
+				preset_base_color = Color(0.15, 0.15, 0.15);
```

Comment:
```suggestion
				preset_base_color = Color(0.12, 0.12, 0.12);
```
Keeping it slightly darker seems better to me.